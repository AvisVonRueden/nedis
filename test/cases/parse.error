asdfasdfasdf*2
dasd$#$---$3
foo
