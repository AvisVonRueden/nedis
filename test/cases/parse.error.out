-ERR Protocol error: number expect after *
